compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -41,7 +37,7 @@ versionDev :: Version
 versionDev = V1 PointDev
 
 supportedInputVersions :: [Version]
-supportedInputVersions = [version1_3, version1_4, version1_5, versionDev]
+supportedInputVersions = [version1_4, version1_5, versionDev]
```

Comment:
I think older versions (including 1.2) are still supported as _input_ with no changes, but it's the `supportedOutputVersions` below instead that is shrinking with these changes, right?

Reply:
Currently, all packages in an application need to be compiled using the same version of the compiler. Hence, there's no point in supporting versions for input that cannot be output.